spec/integration/spec_file_load_errors_spec.rb

```diff
@@ -82,6 +82,26 @@
     EOS
   end
 
+  it 'prints a warning when a helper file exits early' do
+    write_file_formatted "helper_with_exit.rb", "exit 999"
+
+    expect {
+      run_command "--require ./helper_with_exit.rb"
+    }.to raise_error(SystemExit)
+    output = normalize_durations(last_cmd_stdout)
+    expect(output).to eq unindent(<<-EOS)
+
+      An exit occurred while loading ./helper_with_exit.rb.
+      Failure/Error: exit 999
+
+      SystemExit:
+        exit
+      # ./helper_with_exit.rb:1:in `exit'
+      # ./helper_with_exit.rb:1#{spec_line_suffix}
+      No examples found.
```

Comment:
I wonder if we can supress this, maybe setting a global `rspec_is_quitting` in config and gateing this message around that?